### PR TITLE
Added Schedule Page

### PR DESCRIPTION
**Pull request by sarj21**
Waiting on assets so some things will change

### CODE REVIEW

File: src/components/ScheduleSection.vue [hunk 1]
```diff
@@ -1,21 +1,345 @@
...
+    </div>
   </div>
 </template>
 
 <script>
 export default {
   name: 'ScheduleSection',
+  data() {
+    return {
+      events: [
```
Comment (mecolmg) [inline src/components/ScheduleSection.vue]: Nice use of Vue :)

File: src/components/ScheduleSection.vue [hunk 2]
```diff
@@ -1,21 +1,345 @@
...
+          color: $neon-pink;
+          font-size: 15px;
+        }
+        .title {
+          margin-top: -2px;
+          font-size: 20px;
+          color: grey;
+        }
+      }
+
```
Comment (mecolmg) [inline src/components/ScheduleSection.vue]: Too much whitespace

File: src/components/ScheduleSection.vue [hunk 3]
```diff
@@ -1,21 +1,345 @@
...
+    80% { opacity: 0.5; }
+    85% { opacity: 0; }
+    90% { opacity: 0.5; }
+    95% { opacity: 0; }
+    100% { opacity: 0.5; }
+  }
+
+  /* This probably will be going soon */
+  $font-size: 50px;
+  @import url('https://fonts.googleapis.com/css?family=Titillium+Web:900');
```
Comment (mecolmg) [inline src/components/ScheduleSection.vue]: If we do keep this, might wanna add it to vars

Comment (mecolmg) [reply src/components/ScheduleSection.vue]: Also AHHHHH so many fonts 😢

Comment (sarj21) [reply src/components/ScheduleSection.vue]: Probably not keeping it and going towards assets later

File: src/components/ScheduleSection.vue [hunk 4]
```diff
@@ -249,29 +249,29 @@ export default {
...
-    80% { opacity: 0; }
-    85% { opacity: 0.95; }
-    90% { opacity: 0; }
-    95% { opacity: 0.95; }
-    100% { opacity: 0;}
+    60% { opacity: 0; }
+    65% { opacity: 0.95; }
+    70% { opacity: 0.85;}
+    85% { opacity: 0; }
+    100% { opacity: 0.85;}
```
Comment (mecolmg) [inline src/components/ScheduleSection.vue]: Keyframes should end at the same value they begin with to smoothly animate.

**Pull request merged**